.github/workflows/Release.yml

```diff
@@ -7,33 +7,49 @@ jobs:
   build:
     strategy:
       matrix:
-        java: ["8"]
-        distribution: ["temurin"]
+        entry:
+          - { java: "8", distribution: "temurin" }
 
     runs-on: "windows-2019"
 
-    name: Java ${{ matrix.java }} ( ${{ matrix.distribution }} )
+    name: Java ${{ matrix.entry.java }} ( ${{ matrix.entry.distribution }} )
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      # configuring Developer Command Prompt for Microsoft Visual C++
+      # Configuring Developer Command Prompt for Microsoft Visual C++
       - uses: ilammy/msvc-dev-cmd@v1
 
       # Setup JDK
       - uses: actions/setup-java@v3
         with:
-          distribution: ${{ matrix.distribution }}
-          java-version: ${{ matrix.java }}
+          distribution: ${{ matrix.entry.distribution }}
+          java-version: ${{ matrix.entry.java }}
           server-id: ossrh
           server-username: OSSRH_USERNAME
           server-password: OSSRH_PASSWORD
           gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
           gpg-passphrase: MAVEN_GPG_PASSPHRASE
 
-      # Release
-      - run: mvn clean install -P release 
+      # Set git config
+      - run: |

```

Comment:
Consider using a GHA for this, I googled https://github.com/OleksiyRudenko/gha-git-credentials not sure if it's the best.

Reply:
I tried to use it at https://github.com/dblock/log4jna/pull/55 but failed. It may run only on Linux.

Reply:
NBD, thanks for trying